### ISSUE TITLE
'Null' tag?

### ISSUE THREAD

**Issue by derekeder** [datamade/usaddress#80]
In usaddress 0.3.4:

```python
import usaddress
usaddress.tag("728 W. Roosevelt Road (East Side) 2nd floor")
```
```python
> (OrderedDict([('AddressNumber', '728'), ('StreetNamePreDirectional', 'W.'), ('StreetName', 'Roosevelt'), ('StreetNamePostType', 'Road'), ('Null', '(East Side)'), ('OccupancyIdentifier', '2nd'), ('OccupancyType', 'floor')]),
 'Street Address')
```

`Null` is a bit confusing to me. Is this a part of the address we can't identify? Could we call it something else?

**Comment by fgregg** [comment]
So, it's not part of the address. What tag do you suggest.

**Comment by derekeder** [comment]
Perhaps `Unknown` or `NotParsed`?

**Comment by cathydeng** [comment]
'Uncertain' or 'NotAddress' or 'Other'?

**Comment by fgregg** [comment]
"notaddress" :+1:

**Comment by cathydeng** [comment]
closed via 7759223d54dbf7885ffa4e0ba816b6447d1e3e2d
